runtime/lua/vim/lsp.lua

```diff
@@ -1058,7 +1058,7 @@ function lsp.start_client(config)
   --- Returns the default handler if the user hasn't set a custom one.
   ---
   ---@param method (string) LSP method name
-  ---@return function|nil The handler for the given method, if defined, or the default from |vim.lsp.handlers|
+  ---@return lsp-handler|nil The handler for the given method, if defined, or the default from |vim.lsp.handlers|
```

Comment:
Where is this type defined?

Reply:
https://github.com/neovim/neovim/blob/236c20795eb9f11e21e0719b735ea741711acc08/runtime/lua/vim/lsp/types.lua#L3